MultiVersions/Forge/main/resourcify/lang/zh_tw.json

```
@@ -0,0 +1,139 @@
+{
+  "resourcify.screens.close": "關閉",
+
+  "resourcify.type.resource_packs": "資源包",
+  "resourcify.type.data_packs": "資料包",
+  "resourcify.type.shaders": "光影",
+
+  "resourcify.browse.search": "搜尋%s……",
+  "resourcify.browse.sort.relevance": "相關性",
+  "resourcify.browse.sort.downloads": "下載量",
+  "resourcify.browse.sort.follows": "追蹤量",
+  "resourcify.browse.sort.newest": "最近推出",
+  "resourcify.browse.sort.updated": "最近更新",
+  "resourcify.browse.source": "來源",
+  "resourcify.browse.minecraft_version": "Minecraft 版本",
+  "resourcify.browse.minecraft_version.placeholder": "選擇版本",
+  "resourcify.browse.by": "by %s",
+  "resourcify.browse.bisect_ad": "託管您的伺服器在 §b§lBisectHosting§r 並使用 §b§lDIAMOND§r 代碼獲得第一個月 25%% 的折扣。",
+
+  "resourcify.project.description": "描述",
+  "resourcify.project.gallery": "圖庫",
+  "resourcify.project.versions": "版本",
+  "resourcify.project.categories": "類別：",
+  "resourcify.project.external_resources": "外部資源：",
+  "resourcify.project.issues": "議題",
+  "resourcify.project.source": "原始碼",
+  "resourcify.project.wiki": "維基",
+  "resourcify.project.discord": "Discord",
+  "resourcify.project.members": "專案成員：",
+  "resourcify.project.dependencies": "依賴",
+  "resourcify.project.required_dependency": "需求依賴",
+  "resourcify.project.optional_dependency": "可選依賴",
+
+  "resourcify.version.install": "安裝",
+  "resourcify.version.install_version": "安裝 %s",
+  "resourcify.version.installed": "已安裝",
+  "resourcify.version.installing": "安裝中……",
+  "resourcify.version.type.release": "發佈版",
+  "resourcify.version.type.snapshot": "快照版",
+  "resourcify.version.type.beta": "測試版",
```

之前在 RPMTW 做得投票，看要不要改
```suggestion
  "resourcify.version.type.beta": "Beta 版",
```